### PR TITLE
Support KHR_materials_emissive_stength in glTF export

### PR DESCRIPTION
**Pull request by tboggs300**
This PR adds support for the glTF extension KHR_materials_emissive_strength to be exported.

Model used for testing: [link](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/EmissiveStrengthTest)

### CODE REVIEW

**Comment by sebavan** [comment]
This would only work to reexport some previously setup strength.

I wonder if we should not extract the strength from some kind of normalization factor instead ? 

@bghgary any idea ?

**Comment by tboggs300** [comment]
@sebavan Would it be better to add some functions that give access to the value? A setter that is used on load and when used elsewhere, rescales the emissiveColor of the material. This way we can still maintain checking if it is undefined.

**Comment by bghgary** [comment]
I think we should be checking to see if `emissiveColor` values are greater than `1.0`. If the values are less than `1.0`, there is no reason to export the `KHR_materials_emissive_strength` extension.

See the loader implementation here: https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/loaders/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts#L64

For some context, this extension exists because the core glTF spec limited the emissive values between `0.0` and `1.0` failing to consider HDR values.

**Comment by tboggs300** [comment]
Thanks @bghgary 
I'll get to making the change in the code for that implementation soon.

**Review by sebavan — APPROVED** [review]
LGTM !!! Let s wait on @bghgary final review

File: packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts
Comment (sebavan) [inline packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts:3]: should be import type { Material } from 'core/Materials/material';

File: packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts
Comment (sebavan) [inline packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts:4]: should be import { PBRMaterial } from 'core/Materials/pbrMaterial'; basically alway targetting an individual file

File: packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts
Comment (sebavan) [inline packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts:5]: This should not be used

**Review by sebavan — CHANGES_REQUESTED** [review]

File: packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts
Comment (tboggs300) [reply packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts]: Updated

File: packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts
Comment (tboggs300) [reply packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts]: Updated

File: packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts
Comment (tboggs300) [reply packages/dev/serializers/src/glTF/2.0/Extensions/KHR_materials_emissive_strength.ts]: Removed. Was unused.

**Review by sebavan — APPROVED** [review]

**Comment by sebavan** [comment]
@tboggs300 would be great if you could fix the lint/format issue so that we can merge it :-)

You can use `npm run lint:fix` and `npm run format:fix`

**Comment by tboggs300** [comment]
> You can use `npm run lint:fix` and `npm run format:fix`

@sebavan This gave me 650+ changes so I only committed the one on my added file.

**Comment by sebavan** [comment]
> > You can use `npm run lint:fix` and `npm run format:fix`
> 
> @sebavan This gave me 650+ changes so I only committed the one on my added file.

w000t @RaananW not sure it is expected :-) @tboggs300 thanks a lot

**Review by bghgary — APPROVED** [review]
LGTM except a super minor suggestion

**Pull request merged**
